lightning/src/chain/channelmonitor.rs

```diff
@@ -60,7 +64,7 @@ use std::io::Error;
 #[derive(Clone)]
 #[must_use]
 pub struct ChannelMonitorUpdate {
-	pub(super) updates: Vec<ChannelMonitorUpdateStep>,
+	pub(crate) updates: Vec<ChannelMonitorUpdateStep>,
```

Comment:
Is this the final visibility? Or is the final module structure going to encapsulate this again?

Reply:
I'm not certain what the final visibility will look like. This is used by `channel.rs`, so it needs to be `pub(crate)` for now and likely foreseeably so. We may be able to provide an interface that encapsulates it again, however.